### DISCUSSION TITLE
【2023.5.25】community meeting notes

### DISCUSSION THREAD

**Discussion by cubxxw** [openimsdk/open-im-server#419] [meeting]
**Outline of the meeting:**
- [user case studies](https://github.com/OpenIMSDK/community/blob/main/ADOPTERS.md)
- [Register the use of OpenIM](https://github.com/OpenIMSDK/Open-IM-Server/issues/379)
- This section describes the OpenIMSDK project
- What are we doing now
- Overall project planning and [roadmap for OpenIM](https://github.com/orgs/OpenIMSDK/projects/4/views/2)
- We also have Slack channels for you to communicate and discuss. To join, visit https://slack.com/ and join our [👀 Open-IM-Server slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) team channel.
- [Contribution guide](https://github.com/OpenIMSDK/Open-IM-Server/blob/main/CONTRIBUTING.md), describes the entire contribution step
- External developers attract this, and maintainer management planning needs to let external developers know
- Iterations in functionality build barriers that require more features to attract developers.
- Summary of the meeting, [summary of the document](https://docs.google.com/document/d/1nx8MDpuG74NASx081JcCpxPgDITNTpIIos0DS6Vr9GU/edit?usp=sharing)

**Comment by cubxxw** [comment]
#384 File progress for the new version of OpenIM

**Comment by cubxxw** [comment]
https://github.com/orgs/OpenIMSDK/projects/4 
To resolve and allocate some existing issues and some issues that are currently available for external developers to contribute.

**Comment by cubxxw** [comment]
OpenIM Architecture Design: From Client to Server Overall Design. @FGadvancer

**Comment by cubxxw** [comment]
[Feature]: The open im flutter sdk supports the web/pc side #386

**Comment by cubxxw** [comment]
Feature:Deploy Server-side Through a Visual Web Interface. #420
@skiffer-git

**Comment by cubxxw** [comment]
Feature: jssdk server optimize #422

**Comment by cubxxw** [comment]
Feature: data verification during server restart #423

**Reply by Stone-afk** [reply]
1
